blog/arquillian-cube-1.13.0.textile

```diff
@@ -0,0 +1,39 @@
+---
+date: 2017-12-18
+author: lordofthejars
+---
+
+p. We are delighted to announce a new version of Arquillian Cube 1.13.0 with new awesome features in Docker part and bug fixes.
+
+p. It is *important* to note that if you are using @env.dependencies@ or @wait.for.service.list@ properties, we have changed the separation char from white-space to comma-separated. To migrate to new version, just add a comma instead of white space in the property to separate each of the element.
+
+h3. What's new
+
+p. <i class="icon-star"></i> **Use comma to separate multiple values in arquillian.xml**
```

Comment:
We mention this in the opening section - maybe move it to the bottom, so other new features come first to read about?